### PR TITLE
Bump Parquet and Expose Configuration Variables

### PR DESCRIPTION
**Pull request by thegeebe**
See discussion at #328 

Exposes row group size, page size, dictionary encoding, and validation as configurable options to pass to the parquet writer. Bumps Parquet to 1.9.0 which implements more memory efficient dictionary encoding, among other enhancements.

### CODE REVIEW

**Comment by HenryCaiHaiying** [comment]
Looks good overall.  Can you add the 4 new parameters into secor.common.properties with some docs.

**Comment by thegeebe** [comment]
Alright, I put in some comments and made the default values in the config the default values from ParquetWriter implementation; not sure what your convention is for params that have defaults defined in the code. Also I squashed the commits for cleanliness

File: src/main/config/secor.common.properties [hunk 1]
```diff
@@ -381,3 +381,20 @@ secor.file.age.youngest=true
...
+# fine grained reading but have higher space overhead. Defaults to 1MB in Parquet 1.9.
+parquet.page.size=1048576
+
+# Enable or disable dictionary encoding for Parquet writers. The dictionary encoding builds a dictionary of values
+# encountered in a given column. Defaults to true in Parquet 1.9.
+parquet.enable.dictionary=true
+
+# Enable or disable validation for Parquet writers. Validates records written against the schema. Defaults to false in
+# Parquet 1.9.
+parquet.validation=false
```
Comment (HenryCaiHaiying) [inline src/main/config/secor.common.properties]: Looks good to me except there is one extra character at the end of line.

Comment (thegeebe) [inline src/main/config/secor.common.properties]: it's not a new character, just a notification that there is no newline. I added a new line and squashed

**Comment by HenryCaiHaiying** [comment]
Thanks for the investigation and fix, this is a great contribution.

**Pull request merged**